samples/nrf9160/aws_fota/src/main.c

```diff
@@ -33,8 +33,8 @@ BUILD_ASSERT_MSG(!IS_ENABLED(CONFIG_LTE_AUTO_INIT_AND_CONNECT),
 #endif
 
 #if !defined(CONFIG_CLOUD_CLIENT_ID)
-#define IMEI_LEN 15
-#define CLIENT_ID_LEN (IMEI_LEN + 4)
+#define IMEI_LEN 19
```

Comment:
Could we add a comment here document why it has this value?

Reply:
Sure I'll add a comment explaining that it's the length required for the AT CMD response buffer.